src/vm/docs/mod.rs

```diff
@@ -231,6 +231,29 @@ created by this set of bindings is used for evaluating and return the value of `
     example: "(let ((a 2) (b (+ 5 6 7))) (+ a b)) ;; Returns 20"
 };
 
```

Comment:
@lgalabru I had a discussion with @kantai about putting the `define` functions into our function reference.  Can you add `define-data-var` here? I don't see it.

Reply:
Addressed with https://github.com/blockstack/blockstack-core/pull/1022/commits/e6383782b93c81d04ad8910599b4dda70ec60cb3